<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="central_grid"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
   </widget>
   <widget class="QMenu" name="menu_preferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menu_theme">
     <property name="title">
      <string>Theme</string>
     </property>
    </widget>
    <widget class="QMenu" name="menu_font_size">
     <property name="title">
      <string>Font Size</string>
     </property>
    </widget>
    <addaction name="menu_theme"/>
    <addaction name="menu_font_size"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_user_guide"/>
    <addaction name="action_contact_support"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_preferences"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_new">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout_OptiCORD">
   <property name="text">
    <string>About OptiCORD</string>
   </property>
  </action>
  <action name="action_user_guide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="action_contact_support">
   <property name="text">
    <string>Contact Support...</string>
   </property>
  </action>
  <action name="actiong">
   <property name="text">
    <string>g</string>
   </property>
  </action>
  <action name="action_small">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
  </action>
  <action name="action_medium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="action_large">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
